cpp/test/ios/controller/C++ Test Controller.xcodeproj/project.pbxproj

```diff
@@ -371,21 +371,20 @@
 				DEVELOPMENT_TEAM = U4TBVKNQ7F;
 				ENABLE_BITCODE = NO;
 				GCC_INLINES_ARE_PRIVATE_EXTERN = NO;
-				GCC_PREPROCESSOR_DEFINITIONS = (
-					"DEBUG=1",
-				);
+				GCC_PREPROCESSOR_DEFINITIONS = "DEBUG=1";
 				GCC_SYMBOLS_PRIVATE_EXTERN = NO;
 				HEADER_SEARCH_PATHS = (
 					../../include,
-					"../../../../sdk/$(PLATFORM_NAME).sdk/usr/include",
+					../../../include,
+					../../../include/generated,
 				);
 				INFOPLIST_FILE = Bundle/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Bundles";
-				LIBRARY_SEARCH_PATHS = "../../../../sdk/$(PLATFORM_NAME).sdk/usr/lib";
+				LIBRARY_SEARCH_PATHS = "../../../lib/$(PLATFORM_NAME)";
```

Comment:
we link directly with the static libraries from `lib/iphoneos` or `lib/iphonesimulator`.